numba/cuda/stubs.py

```diff
@@ -399,6 +399,26 @@ class sub(Stub):
         atomically.
         """
 
+    class and_(Stub):
+        """and_(ary, idx, val)
+
+        Perform atomic ary[idx] &= val. Supported on int32, int64, uint32 and
+        uint64 operands only.
+
+        Returns the old value at the index location as if it is loaded
+        atomically.
+        """
+
+    class or_(Stub):
+        """or_(ary, idx, val)
+
+        Perform atomic ary[idx] \|= val. Supported on int32, int64, uint32 and
+        uint64 operands only.
+
+        Returns the old value at the index location as if it is loaded
+        atomically.
+        """
```

Comment:
I hadn't noticed that escaping the pipe symbol for Sphinx would cause a flake8 error - I can't think of another way to escape this though, so perhaps the best thing to do here is to suppress that particular warning for this docstring with:

```suggestion
        """  # noqa: W605
```